site/docs/4.8.0/overview/releaseNotes.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Apache BookKeeper 4.8.0 Release Notes
+---
+
+This is the 13th release of Apache BookKeeper!
+
+The 4.8.0 release incorporates hundreds of bug fixes, improvements, and features since previous major release, 4.7.0.
+It is a new big milestone in Apache BookKeeper community - after merging Yahoo and Saleforce branches
```

Comment:
Not sure we need "after merging Yahoo and Salesforce branches". isn't that already done in previous releases?

Reply:
dropped